bolt/test/runtime/instrument-wrong-target.s

```diff
@@ -1,7 +1,8 @@
 # Test that BOLT errs when trying to instrument a binary with a different
 # architecture than the one BOLT is built for.
 
-# REQUIRES: x86_64-linux,bolt-runtime,target=x86_64{{.*}}
+# REQUIRES: x86_64-linux,bolt-runtime
+# REQUIRES: target-x86_64 && aarch64-registered-target
```

Comment:
I've never seen this syntax with `&&`. Is it a new addition? Anyway, if you tested it and it works, LG.